custom/up_nrhm/sql_data.py

```diff
@@ -198,3 +198,89 @@ def filters(self):
     @property
     def group_by(self):
         return []
+
+
+class ASHAFunctionalityChecklistData(SqlData):
+    slug = 'asha_functionality_checklist'
+
+    @property
+    def table_name(self):
+        return get_table_name(self.config['domain'], TABLE_ID)
+
+    def _qualify_column(self, column):
+        return '"{}".{}'.format(self.table_name, column)
+
+    @property
+    def columns(self):
+        return [
+            DatabaseColumn("Total number of ASHAs under the Facilitator", SimpleColumn("case_id",)),
```

Comment:
There is a `doc_id` column which you could use which wouldn't require any grouping and could then also serve as the filter in `ASHAAFChecklistData`.